### PR TITLE
Add im.bernard.Memorado

### PR DESCRIPTION
**Pull request by wbernard**
### Please confirm your submission meets all the criteria

Learn using spaced repetition. Create cards with a question and answer, and practice with them.

- [X] Please describe your application briefly.
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project.
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned.

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Comment by wbernard** [comment]
bot, build

**Comment by wbernard** [comment]
bot, build

File: im.bernard.Memorado.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+    	{
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
```
Comment (hfiguiere) [inline im.bernard.Memorado.json:27]: this need the cleanup rule to remove it after build.

File: im.bernard.Memorado.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag": "v0.8.1"
+                }
+        	]
+        },
+        {
+            "name" : "memorado",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "config-opts" : ["-Dprofile=development"],
```
Comment (hfiguiere) [inline im.bernard.Memorado.json:40]: I doubt this is the correct profile. This is not a development package.

**Comment by hfiguiere** [comment]
bot, build im.bernard.Memorado

File: im.bernard.Memorado.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    	{
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag": "v0.8.1"
+                }
+        	]
```
Comment (A6GibKm) [inline im.bernard.Memorado.json:34]: nit: Indentation seems off here.

File: im.bernard.Memorado.json [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        },
+        {
+            "name" : "memorado",
+            "builddir" : true,
```
Comment (A6GibKm) [inline im.bernard.Memorado.json:38]: I don't think this is needed here.

File: im.bernard.Memorado.json [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    ],
+    "modules" : [
+    	{
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag": "v0.8.1"
```
Comment (A6GibKm) [inline im.bernard.Memorado.json:32]: Current is v0.10.0. If the app works with it, it should use it.

File: im.bernard.Memorado.json (hunk 1 above)
Comment (A6GibKm) [reply im.bernard.Memorado.json:27]: See https://github.com/flathub/re.sonny.Tangram/blob/master/re.sonny.Tangram.json#L33

Comment (wbernard) [reply im.bernard.Memorado.json:27]: Should be fixed now

File: im.bernard.Memorado.json (hunk 4 above)
Comment (wbernard) [reply im.bernard.Memorado.json]: Removed :+1:

File: im.bernard.Memorado.json (hunk 5 above)
Comment (wbernard) [reply im.bernard.Memorado.json]: Thanks, updated :+1:

**Comment by wbernard** [comment]
bot, build im.bernard.Memorado

**Comment by wbernard** [comment]
bot, build im.bernard.Memorado

File: im.bernard.Memorado.json [hunk 6]
```diff
@@ -42,7 +42,7 @@
                     "type" : "git",
                     "url" : "https://github.com/wbernard/Memorado.git",
                     "tag" : "0.1",
-                    "commit" : "f5f9b1770feefaf003994328ce2b7e4483071fc7"
+                    "commit" : "52bdfcefe4352c9f49f0b496f088a1bc07e16f63"
```
Comment (hfiguiere) [inline im.bernard.Memorado.json:45]: retagging IS a bad practice. This should have been a new tag.

Comment (bertob) [reply im.bernard.Memorado.json:45]: Ah, including for the very first release?

Comment (bbhtt) [reply im.bernard.Memorado.json:45]: Yes

File: im.bernard.Memorado.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
+{
+    "id" : "im.bernard.Memorado",
```
Comment (bbhtt) [inline im.bernard.Memorado.json:2]: How to verify that domain belongs to the project here? The GitHub repo doesn't have a link to it, the page has nothing in it.

Comment (bertob) [reply im.bernard.Memorado.json:2]: Added a URL to the Github page for now, I'll make a little website for it later.

Comment (bertob) [reply im.bernard.Memorado.json:2]: Update: There's now a website here http://bernard.im/memorado

Comment (bbhtt) [reply im.bernard.Memorado.json:2]: Thanks

**Pull request closed without merging**